### PR TITLE
Added Image component

### PR DESCRIPTION
**Pull request by alexpricedev**
Screenshot 2019-11-25 at 20 02 53
Screenshot 2019-11-25 at 20 02 51
Screenshot 2019-11-25 at 20 02 47
Screenshot 2019-11-25 at 20 02 44

### CODE REVIEW

File: packages/open-ui/src/components/Image/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { MarginStyleKeys } from 'interfaces/MarginStyleKeys';
+import { BorderRadiusOption } from 'interfaces/BorderRadiusOption';
+import { getMarginProps } from 'utils/helpers';
+import styles from './Image.module.scss';
+import { toSentenceCase } from 'utils/strings/to-sentence-case';
+
+interface ImageProps {
```
Comment (gurjit03) [inline packages/open-ui/src/components/Image/Image.tsx:10]: @remotealex should we be using interface here ? Types seems relevant aint ? or where should we use Types ?

File: packages/open-ui/src/interfaces/BorderRadiusOption.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export type BorderRadiusOption = 'small' | 'medium' | 'large';
```
Comment (gurjit03) [inline packages/open-ui/src/interfaces/BorderRadiusOption.ts:1]: @remotealex we are exporting Types but naming them as interfaces :thinking: ?

**Review by gurjit03 — CHANGES_REQUESTED** [review]

File: packages/open-ui/src/components/Image/Image.tsx (hunk 1 above)
Comment (alexpricedev) [reply packages/open-ui/src/components/Image/Image.tsx:10]: I think CRA warns to use interfaces when they are _object_ shaped. Types seem to be for _value_ based types.

File: packages/open-ui/src/interfaces/BorderRadiusOption.ts (hunk 2 above)
Comment (alexpricedev) [reply packages/open-ui/src/interfaces/BorderRadiusOption.ts:1]: Yeah, i think thats fine. They're basically the same :)

File: packages/open-ui/src/components/Image/Image.tsx (hunk 1 above)
Comment (alexpricedev) [reply packages/open-ui/src/components/Image/Image.tsx:10]: Oh yeah! and you can `extend` interfaces which is handy for this sometimes

Comment (gurjit03) [reply packages/open-ui/src/components/Image/Image.tsx:10]: value based meaning like `type = 'somevalue' | 'somevalue2'` ?

Comment (remotealex) [reply packages/open-ui/src/components/Image/Image.tsx]: Oh yeah! and you can `extend` interfaces which is handy for this sometimes

File: packages/open-ui/src/interfaces/BorderRadiusOption.ts (hunk 2 above)
Comment (gurjit03) [reply packages/open-ui/src/interfaces/BorderRadiusOption.ts:1]: fair then :)

**Pull request merged**